test/io-tests/configs/defaults.config

```diff
@@ -1,3 +1,4 @@
 db-uri = "required"
 db-schemas = "required"
 db-anon-role = "required"
+db-load-guc-config = false
```

Comment:
```suggestion
# Not the default, but only works with proper db-uri
db-load-guc-config = false
```

Reply:
The tests fail by adding this line, I'd need to strip the comment for going green. Do you think it's worth it?

Reply:
How so? I made the change in the config file - not the expected output file. I would expect the output to stay the same?

Reply:
Ah, my bad. You're right. I'll add the change.